docs/docs/guides/how-to-join-a-competition.md

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Join a Competition'
+slug: /guides/how-to-join-a-competition
+sidebar_position: 2
+---
+
+### Overview
+Competitions can be hosted by users or groups. Participating in these competitions can be a fun way to progress your account. This guide explains how participation works for both user-created and group-created competitions.
+
+### Prerequisites
+- Your player profile exists on Wise Old Man (opted into tracking). See: [`/guides/how-to-start-tracking`](/guides/how-to-start-tracking)
+
+### Competition Types
+
+#### User-created competitions
+- Typically created for a group of friends who don't necessarily belong to the same group (clan) on the website.
+- Only the host can add participants - it is not possible to view who the host is.
+- There is not an on-site Join or Request to Join feature.
+
+#### Group-created competitions
+- Created by a group’s leaders/admins.
+- Typically restricted to group members.
+- Only group leaders can add participants (usually from their group roster).
+- You may need to contact a group leader to be added.
+
+### How to participate
+
+1. Ensure your profile exists and is up to date
+   1. If you’ve never been tracked, follow the Start Tracking guide to create your profile.
+   2. Consider requesting a fresh update so the host can validate your stats.
+   
+2. Find an active competition
+   1. Browse `https://wiseoldman.net/competitions` and filter by metric, status, and/or type.
```

Comment:
Making links clickable might be easier for navigation so you don't have to copy paste the link manually.